src/fides/api/ctl/database/samples.py

```diff
@@ -0,0 +1,163 @@
+"""
+Sample data to use for seeding the database. This are loaded & parsed from the
+sample project at src/fides/data/sample_project.
+
+See load_samples() in seed.py for usage.
+"""
+from typing import Dict, List, Optional, TextIO, Union
+
+import yaml
+from expandvars import expandvars  # type: ignore
+from fideslang.models import Dataset, Organization, Policy, System
+from fideslang.validation import FidesKey
+
+# TODO: is there a way to use the stdlib in Python 3.9+?
+# from importlib.resources import files
+from importlib_resources import files
```

Comment:
@ThomasLaPiana is there a better way to do this? ie. use a backported lib in Python 3.8, but the stdlib in newer versions... or do we just need to "remember" to remove this later

Reply:
this seems fine to me. Are you talking about instead something like a try/except where you try importing files from importlib.resources first and the except gets it from importlib_resources? Or conditionally importing from one or the other based on the version?

Reply:
Yeah, something like that feels slightly better... I dunno, I think I'll just accept it. I'm overengineering aren't I